app/components/status_tags/requirements_component.rb

```diff
@@ -2,17 +2,22 @@
 
 module StatusTags
   class RequirementsComponent < StatusTags::BaseComponent
-    def initialize(requirements:)
-      @requirements = requirements
+    def initialize(planning_application:, requirements:)
+      @planning_application = planning_application
+      @requirements = @planning_application.requirements
```

Comment:
It looks like the requirements parameter is never actually used any more, so probably safe to remove